### DISCUSSION TITLE
Blur effect

### DISCUSSION THREAD

**Discussion by nielsbosma** [AvaloniaUI/Avalonia#9100] [Q&A]
Is there any way to create a blur effect where anything in a Panel is blurred like this:

![image](https://user-images.githubusercontent.com/4034492/193882526-436cee68-cf88-488a-b778-50809ac7e806.png)

**Comment by timunie** [comment]
I think Aura.UI has something similar. Maybe you can have a look how they did it. https://github.com/PieroCastillo/Aura.UI

**Comment by kekekeks** [comment]
Planned for either 11.0 or 11.1.

**Comment by maxkatz6** [comment]
It also can be done with Skia filters:
https://gist.github.com/kekekeks/ac06098a74fe87d49a9ff9ea37fa67bc

**Reply by nielsbosma** [reply]
Cool thanks a lot!
